../../../pyodbc.pyi,sha256=R-lJ63bwP_7ETJL2QFphFJQCCB2tzbon0SBfzNDkBPw,13165
pyodbc-4.0.35.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
pyodbc-4.0.35.dist-info/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
pyodbc-4.0.35.dist-info/METADATA,sha256=eWhyIxFGUQbFUvv_3pG4rhPPywBsNU7TlH2Pf9Yvfqg,2750
pyodbc-4.0.35.dist-info/RECORD,,
pyodbc-4.0.35.dist-info/REQUESTED,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
pyodbc-4.0.35.dist-info/WHEEL,sha256=nKSwEH5fkxvG0Vdj1Hx7vbuU-SGQ9Nxl4yFFsCilvhs,152
pyodbc-4.0.35.dist-info/top_level.txt,sha256=30jvYcCul6_wA26OT5c068GAHzqQLpd9WfQ0mSSsDTU,7
pyodbc.cpython-310-x86_64-linux-gnu.so,sha256=q9NRgSljc8hDWPje8LWikDo0pwX5Ui1vzs0s5tCwtH0,919840
